test/utils.py

```diff
@@ -864,21 +864,14 @@ def frequent_thread_switches():
     """Make concurrency bugs more likely to manifest."""
     interval = None
     if not sys.platform.startswith('java'):
```

Comment:
You can remove the 'java' if statement altogether because we don't test Jython anymore.

Reply:
Done